src/sentry/utils/snowflake.py

```diff
@@ -1,9 +1,18 @@
+import random
+import secrets
 from dataclasses import dataclass
-from datetime import datetime
-from typing import OrderedDict
+from datetime import datetime, timedelta
+from typing import List, OrderedDict, Tuple
 
 from django.conf import settings
 
+from sentry.utils import json, redis
+
+# come back to this, this might be called more than once
+BASE_SEED = secrets.randbits(64)
```

Comment:
This could use a docstring to explain what it's doing. That is: we want `get_region_sequence_assignment_order` to produce the same order if called twice for the same timestamp, but if there are multiple server nodes doing the same thing concurrently, we want to make sure they are seeded differently.

Also, the whole thing about producing the same order for the same timestamp doesn't really matter until we're caching locally before calling out to Redis. It occurred to me today, it might make sense to create one class (probably a global singleton?) that holds both the cache and the base seed. But we can come back to that later.